demo-app/src/CreateShape.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from "react";
+import ClipPathPreviewForm from "./ClipPathPreviewForm.js";
+import ClipPathPreview from "./ClipPathPreview.js";
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+const CreateShape = (props) => {
+  
+  // Change these keys to the ones in schema.js
+  const [formInputs, setFormInputs] = useState({
+    //Schema Variables
+    "name": "Square", 
+    "type": "square",
+    "formula": "polygon(10% 10%, 90% 10%, 90% 90%, 10% 90%)",
+    "vertices": "4",
+    "edges": "4", 
+    "notes": "A circle is a round shaped figure that has no corners or edges. In geometry, a circle can be defined as a closed, two-dimensional curved shape.", 
+    //Other Variables
+    "clipPathType": "polygon",
+    "showShadow": false, 
+    "backgroundColor": "#12a8d6",
+  });
+
+  function handleChange(event) {
+
+    const name = event.target.name;
+
+    const value = event.target.type === "checkbox" ? event.target.checked : event.target.value;
+
+    if (name === "name") {
+      setFormInputs({
+        ...formInputs, 
+        "name": value, 
+        "type": value.toLowerCase(),
+      });
+    } else if (name === "formula") {
+
+      const edgeVerticeNumber = formInputs.clipPathType === "polygon" ? value.split(",").length: 0;
+
+      if (value.includes("(") && value.includes(")")) {
+        if (value.includes("polygon")) {
+          setFormInputs({
+            ...formInputs, 
+            "formula": value, 
+            "clipPathType": "polygon",
+            "vertices": edgeVerticeNumber, 
+            "edges": edgeVerticeNumber, 
+          });
+        } else if (value.includes("circle")) {
+          setFormInputs({
+            ...formInputs, 
+            "formula": value, 
+            "clipPathType": "circle",
+            "vertices": edgeVerticeNumber, 
+            "edges": edgeVerticeNumber, 
+          });
+        } else if (value.includes("ellipse")) {
+          setFormInputs({
+            ...formInputs, 
+            "formula": value, 
+            "clipPathType": "ellipse",
+            "vertices": edgeVerticeNumber, 
+            "edges": edgeVerticeNumber, 
+          });
+        } else setFormInputs({
+          ...formInputs, 
+          "formula": value, 
+          "vertices": edgeVerticeNumber, 
+          "edges": edgeVerticeNumber,
+        });
+      }
+    } else if (name === "clipPathType") {
+      if (value === "polygon") {
+        setFormInputs({
+          ...formInputs, 
+          "clipPathType": value, 
+          "name": "Square", 
+          "type": "square", 
+          "edges": 4,
+          "vertices": 4,
+          "formula": "polygon(10% 10%, 90% 10%, 90% 90%, 10% 90%)", 
+          "notes": "A square is closed, two-dimensional shape with 4 equal sides. A square is a quadrilateral.", 
+        });
```

Comment:
name, notes, and formula like the initial object creation

`name`: Tilted Square 
`notes`  as empty(`''`). 
Make the formula as, `polygon(10% 10%, 90% 10%, 90% 90%, 10% 80%)`.

Reply:
Changed the preset values. Do you want me to change the preset name and formula for the circle and ellipse at all?

Reply:
done